src/components/AddItem.js

```diff
@@ -0,0 +1,9 @@
+const AddItem = () => {
+  return (
+    <div>
+      <h1>Add Item</h1>
+    </div>
+  );
```

Comment:
```suggestion
return <h1>Item List</h1>;
```

The div nesting here isn't hurting anything but is a common anti-pattern since it's adding unnecessary elements to the DOM. Since the only element here is the `<h1>` we can omit it. If we need to return multiple elements in a single component, React has [Fragments](https://reactjs.org/docs/fragments.html#short-syntax) that can be used to encapsulate without adding an additional nested `<div>` or other container element.

One of the motivations for maintaining [Semantic HTML](https://blog.hubspot.com/website/semantic-html) (using tags that convey meaning) is accessibility, which we'll talk about later in a Learning Module!

Reply:
Very helpful info here.